src/parser/monk/mistweaver/modules/talents/Lifecycles.tsx

```diff
@@ -60,8 +60,8 @@ class Lifecycles extends Analyzer {
       this.castsNonRedViv += 1;
       return;
     }
-    this.manaSaved += SPELLS.VIVIFY.manaCost * LIFECYCLES_MANA_PERC_REDUCTION;
-    this.manaSavedViv += SPELLS.VIVIFY.manaCost * LIFECYCLES_MANA_PERC_REDUCTION;
+    this.manaSaved += (SPELLS.VIVIFY.manaCost ? SPELLS.VIVIFY.manaCost : 1) * LIFECYCLES_MANA_PERC_REDUCTION;
```

Comment:
If these values are ever null and it defaults to 1 it will just provide inaccurate information making the whole statistic pointless and misleading. Since it wouldn't crash people won't come ask for it to be fixed either and just assume `damn lifecycles is literally the worst talent in the game and even playing it to the mas it doesn't save me more than 100 mana` on top of then maybe going into peak and making me have to deal with them there